### PR TITLE
adds default env setups for various commands

### PR DESCRIPTION
**Pull request by jonathancreamer-eb**

### CODE REVIEW

File: packages/brite-core/src/commands/start.ts [hunk 1]
```diff
@@ -12,4 +12,11 @@ export default class BriteStartCommand extends BriteCommand {
     public async execute(): Promise<IBriteCommandResult> {
         return new WebpackPlugin(this.logger, this.options).run('start');
```
Comment (kwelch-eb) [inline packages/brite-core/src/commands/start.ts]: Add call to `defaultEnv`

File: packages/brite-core/src/commands/test.ts [hunk 2]
```diff
@@ -52,4 +52,11 @@ export default class BriteTestCommand extends BriteCommand {
 
         return result;
```
Comment (kwelch-eb) [inline packages/brite-core/src/commands/test.ts]: add call to defaultEnv

File: packages/brite-core/src/__tests__/command.spec.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        };
+    }
+}
+
+describe('Brite Commands', () => {
+    let runner: BriteCommandRunner;
+    const env = process.env.NODE_ENV;
+
+    beforeEach(() => {
+        // Temporarily delete just to make sure we can actually set it
```
Comment (kwelch-eb) [inline packages/brite-core/src/__tests__/command.spec.ts]: Can we add a test that checks default/overriding as well.

File: packages/brite-core/src/commands/index.ts [hunk 4]
```diff
@@ -5,11 +5,11 @@ import { default as test } from './test';
...
 export {
-    IBriteCommand,
-    IBriteCommandResult,
-    IBriteCommandConstructor,
+    BaseBriteCommand,
+    BriteCommandResult as IBriteCommandResult,
+    BriteCommandConstructor as IBriteCommandConstructor,
     IBriteCommandOptions,
-    BriteCommand,
+    BaseBriteCommand as BriteCommand,
```
Comment (kwelch-eb) [inline packages/brite-core/src/commands/index.ts]: Exported twice?

File: packages/brite-core/src/__tests__/command.spec.ts (hunk 3 above)
Comment (jcreamer898) [reply packages/brite-core/src/__tests__/command.spec.ts]: That's what this test does no?

**Pull request merged**